src/zcash/address/bip44.cpp

```diff
@@ -4,6 +4,10 @@
 
 #include "bip44.h"
 
+HDKeyPath libzcash::Bip44TransparentAccountKeyPath(uint32_t bip44CoinType, libzcash::AccountId accountId) {
+    return "m/44'/" + std::to_string(bip44CoinType) + "'/" + std::to_string(accountId) + "'";
+}
```

Comment:
Could you modify `libzcash::Bip44AccountChains::DeriveExternal()` and `libzcash::Bip44AccountChains::DeriveInternal()` to take advantage of this function?

Reply:
Fixed by  075f55427195f507fdcc4b51bd082980b8eb107f